### DISCUSSION TITLE
sf version create --dry-run flag

### DISCUSSION THREAD

**Discussion by dschach** [forcedotcom/cli#2607] [Ideas]
I'd like to do a debug/dry-run of the package version creation command.

`sf package version create --package "My Package" --installation-key-bypass --code-coverage --dry-run`

It would return the usual JSON but would have a placeholder instead of the `PackageVersionId` value. This includes the version number with build, etc.

That way, I can see what is happening so far as I develop my package, but do not have to create a new version until I'm ready.

**Comment by shetzel** [comment]
This is the same discussion as https://github.com/forcedotcom/cli/discussions/2258 so I'm going to move your post there.
